src/executors/protect_executor.rs

```diff
@@ -60,6 +59,9 @@ where
                 .context("Error getting gas price: {}")?;
         }
         action.tx.set_gas_price(bid_gas_price);
+        // set max_priority_fee_per_gas to be 0
+        action.tx.as_eip1559_mut().unwrap().max_priority_fee_per_gas = Some(U256::from(50));
```

Comment:
try to avoid unwraps as much as possible! Since this fn returns a result, we can just use the ? operator here

Reply:
also is there no `set_max_priority_fee_per_gas` similar the above used set_gas_price?

Reply:
also - class constant or config for the magic number

Reply:
ya there is no setter weirdly enough

Reply:
ok doing `if let Some(...)`